spark/dl/src/main/scala/com/intel/analytics/bigdl/example/MLPipeline/DLClassifierLeNet.scala

```diff
@@ -37,7 +37,7 @@ import org.apache.spark.sql.SQLContext
 object DLClassifierLeNet {
 
   LoggerFilter.redirectSparkInfoLogs()
-  Logger.getLogger("com.intel.analytics.bigdl.optim").setLevel(Level.INFO)
+//  Logger.getLogger("com.intel.analytics.bigdl.optim").setLevel(Level.INFO)
```

Comment:
If you want to remove code, remove it. NEVER comment it to remove code.

Reply:
Are you sure you can remove the log level? What's the log looks like when you run the example after remove it?